src/test/scala/cromwell/util/TryUtilSpec.scala

```diff
@@ -13,15 +14,21 @@ import scala.util.Success
 
 class TryUtilSpec extends FlatSpec with Matchers with MockitoSugar {
 
+  class TransientException extends Exception
   class MockWork {
     var counter: Int = _
```

Comment:
I hate vars but I'm not going to gate the PR on removing this one.  Though nobody should be surprised if this mysteriously disappears in the near future. :smile: